services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasksutils.py

```diff
@@ -24,8 +28,14 @@ async def run_before_shutdown_actions(
                     command=user_service_command.command,
                     timeout=user_service_command.timeout,
                 )
-            except ContainerExecContainerNotFoundError:
+
+            except (
+                ContainerExecContainerNotFoundError,
+                ContainerExecCommandFailedError,
+                ContainerExecTimeoutError,
+            ) as e:
                 _logger.warning(
-                    "Could not run before_shutdown commands because container %s was not found",
+                    "Could not run before_shutdown in container %s, because: %s",
                     container_name,
+                    f"{e}",
```

Comment:
MINOR: To show errors, `ruff` recommends to use in this case `exc_info=True` instead , i.e
```python
                _logger.warning(
                    "Could not run before_shutdown in container %s,
                    container_name,
                    exc_info=True
                )